iset.mm

```diff
@@ -21226,9 +21226,10 @@ effectively bound by a quantifier or something that expands to one (such
     $d y z A $.  $d x z $.
     rgen2a.1 $e |- ( ( x e. A /\ y e. A ) -> ph ) $.
     $( Generalization rule for restricted quantification.  Note that ` x ` and
-       ` y ` needn't be distinct (and illustrates the use of ~ dvelimor ).
+       ` y ` are not required to be disjoint.  This proof illustrates the use
+       of ~ dvelim .  Usage of ~ rgen2 instead is highly encouraged.
        (Contributed by NM, 23-Nov-1994.)  (Proof rewritten by Jim Kingdon,
-       1-Jun-2018.) $)
+       1-Jun-2018.)  (New usage is discouraged.) $)
```

Comment:
Why not also "(Proof modification is discouraged.)" as in set.mm?

Reply:
The proof is totally different from the set.mm proof (unless they are similar by coincidence), so I didn't see any reason to copy that over.